lib/util.ml

```diff
@@ -0,0 +1,27 @@
+module Result = struct
+  module List = struct
+    open Rresult.R.Infix
```

Comment:
if you redefine a few `Result` function you probably don't need to depend on `Rresult` and can inline bind and map directly.